src/app/views/App.tsx

```diff
@@ -339,10 +347,27 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-
+    if (permissionModeType && this.state.hideDialog) {
+      this.showDialog();
+    }
+    const teamsapp = <a href={"https://www.bing.com/?form=000010"}>Sample Explorer Teams app</a>;
```

Comment:
```suggestion
    const teamsapp = <a href={"https://www.bing.com/?form=000010"}>{translateMessage('Sample Explorer Teams app')}</a>;
```

It's important to recognize that, even though the text is a hyperlink, it still needs to be localized so that it translates properly; I have suggested the appropriate changes to make this happen ^^

Reply:
Great catch